closed/get_openssl_source.sh

```diff
@@ -59,8 +59,9 @@ do
 	esac
 done
 
-if [ ${OPENSSL_VERSION:0:4} != "1.1." ]; then
-	usage
+if [ "${OPENSSL_VERSION:0:5}" != "1.0.2" -a "${OPENSSL_VERSION:0:4}" != "1.1." ] ; 
+then
+        usage
```

Comment:
Inconsistent indent: please use a tab as it was before.

Reply:
Done